index.html

```diff
@@ -7,6 +7,11 @@
 
     <link rel="icon" type="image/png" href="./assets/images/icon-sm.png" />
 
+    <!-- Font Face -->
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+
```

Comment:
Good👍

Reply:
You can remove this. The font is no longer being changed, so this is no longer necessary.